py_openda/README_ENKF.md

```diff
@@ -0,0 +1,182 @@
+## Using the main program
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=4902598)